src/LoggedStatement.php

```diff
@@ -35,9 +42,10 @@ public function bindValue(
         int $dataType = PDO::PARAM_STR
     ) : bool
     {
+        /** @var int|string $parameter */
         $result = parent::bindValue($parameter, $value, $dataType);
 
-        if ($result && $this->logEntry !== null) {
+        if ($result && !empty($this->logEntry)) {
```

Comment:
I don't think empty() is quite the same thing as `!== null` -- can you say why the change?

Reply:
The `logEntry` is defined as an array in the constructor so it does not accept nulls, as such it will not be null here. We can change it if you wish but then line 29 will complain (constructor).